anaconda

```diff
@@ -925,56 +925,14 @@ if __name__ == "__main__":
     # make sure we have /var/log soon, some programs fail to start without it
     iutil.mkdirChain("/var/log")
 
-    # Share these with the exit handler, installed later
-    pidfile_path = '/var/run/anaconda.pid'
-    pidfile_created = False
+    # Create a PID file. The exit handler, installed later, will clean it up.
+    import pid
```

Comment:
Cannot agree more! But could we move this import next to the other "standard" (sys, os,...) ones?